conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import cmake_layout, CMakeToolchain, CMakeDeps, CMake
+from conan.tools.env import VirtualBuildEnv
+from conans.errors import ConanInvalidConfiguration
+
+
+class AsmParser(ConanFile):
+    name = "asm-parser"
+    version = "2.0"
```

Comment:
Or whatever it is now

Reply:
0.9 was the latest, so 1.0 should be fine